### DISCUSSION TITLE
FerrisChat v1 Checklist!

### DISCUSSION THREAD

**Discussion by hydrostaticcog** [FerrisChat/FerrisChat#17] [Announcements]
As we near v1, we wanted to let you know the status of the things needed to be completed for v1:

- [x] Functional REST API
- [x] `MESSAGE_CREATE`
- [x] Other events
- [ ] Roles
- [ ] Permissions
- [ ] Working Client 
- [ ] Basic Interactions API
- [ ] Moderation Features
- [ ] Optimization 

@tazz4843 , @randomairborne feel free to DM me if this list needs changing.

**Comment by tazz4843** [comment]
roles are only partly implemented
